ledger/simulation/simulation_eval_test.go

```diff
@@ -151,15 +131,15 @@ func simulationTest(t *testing.T, f func(accounts []simulationtesting.Account, t
 
 	testcase := f(accounts, txnInfo)
 
-	actual, err := s.Simulate(simulation.Request{TxnGroups: [][]transactions.SignedTxn{testcase.input}})
+	actual, err := s.Simulate(testcase.input)
 	require.NoError(t, err)
 
 	validateSimulationResult(t, actual)
 
 	require.Len(t, testcase.expected.TxnGroups, 1, "Test case must expect a single txn group")
-	require.Len(t, testcase.expected.TxnGroups[0].Txns, len(testcase.input), "Test case expected a different number of transactions than its input")
+	require.Len(t, testcase.expected.TxnGroups[0].Txns, len(testcase.input.TxnGroups[0]), "Test case expected a different number of transactions than its input")
```

Comment:
nit: can we slightly improve by 2 parts:
- compare expected txn-groups length == actual txn-groups length. In this case both should be 1
- iterately compare txn-group length expected == actual, so we can remove line 139 later if we support multiple txn-groups?

Reply:
Good suggestions, added in ea3372ea0